### ISSUE TITLE
Database elements not migrated to mysql when trying to deply on ubuntu

### ISSUE THREAD

**Issue by PaulGilmartin** [miguelgrinberg/microblog#128]
Hi,

I've been following this tutorial very closely and have been using it as a guide to build my own site (source code: https://github.com/PaulGilmartin/ownsite). Locally, everything works perfectly and my website accurately displays the database objects I have saved in the local sql lite db. However, i'm running into an issue when trying to deploy my website on an ubuntu server, as described in Chapter 17. It seems that even when i follow the guide for setting up the mysql db, the tables it creates have no elements - I end up with a mysql db with a table for each model I created, but no records in any of the tables. I've exhausted every avenue I can think of to debug this and nothing seems to be working - does anyone have any idea what may be going on here?

Thanks,

Paul

**Comment by miguelgrinberg** [comment]
The database migrations create the tables, they do not add records to those tables. The idea is that the content of the development vs. production databases is not going to be the same, each deployment will generate its own content. Does that make sense?

**Comment by PaulGilmartin** [comment]
Thanks for taking the time  to reply, really appreciate it. 
It does, it makes perfect sense. Shortly after I wrote this comment I thought to myself, why not just add the records to the mysql db directly in the ubuntu server? It worked fine. The thing that got me stuck on it for so long is that the first time I ran my site from ubuntu, it was somehow managing to find all the records in the dev db - I think it was likely because my .env wasn't set up or because I accidentally forgot to ignore the app.db file in my git, not too sure though. Thanks again!
